server.go

```diff
@@ -36,15 +51,18 @@ func main() {
 	dcrstakepoolLog.Infof("Version: %s", version())
 	dcrstakepoolLog.Infof("Network: %s", activeNetParams.Params.Name)
 
-	filename := flag.String("config", "config.toml", "Path to configuration file")
+	// goji uses a bind flag in bind/bind.go, this allows go-flags to parse
+	// despite the conflict
+	flag.CommandLine.Init(os.Args[0], flag.ContinueOnError)
```

Comment:
BTW, anything on that branch you'd want as a PR?
Perhaps [this](https://github.com/chappjc/dcrstakepool/commit/d8ee7e0b540e9519c4ae7daf2bb0e6f3323e133a) or the uslice removal?